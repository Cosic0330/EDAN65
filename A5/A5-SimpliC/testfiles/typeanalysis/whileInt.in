int i() {
    while(1) {
        
    }
}