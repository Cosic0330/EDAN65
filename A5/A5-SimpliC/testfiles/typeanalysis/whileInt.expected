Error at line 2: Found Type: <int>, ExpectedType: <bool>
